### ISSUE TITLE
Hompage does not show any data

### ISSUE THREAD

**Issue by daninden** [dingo35/SmartEVSE-3.5#5]
**Describe the bug**
I updated my SmartEVSE to 3.5.0-RC0, and altough the API endpoints are still working, the homepage stays blank.

It does seem to load some resources.

**To Reproduce**
1. Go to the homepage
2. See nothing

**Expected behavior**
To see the homepage

**Screenshots**
![image](https://github.com/dingo35/SmartEVSE-3.5/assets/7497671/282c0140-8899-4ded-ac63-fce19bd6d20b)

**Comment by daninden** [comment]
I did some quick checks.. looks like the styling.css is not loading properly.

If I save the html page, set the endpoint fixed it is actually working, but only because styling.css has the wrong uri then.
If I fix the uri (include the actual ip endpoint), it starts to fail.

If i load the file directly (http://ip/styling.css) it keeps on waiting, i think the webserver has trouble serving the file (size?).

**Comment by dingo35** [comment]
Cannot reproduce this. Did you reflash spiffs.bin?

**Comment by daninden** [comment]
yes i did.. but I just redid it.. and now it's ok.. very strange.
